aepp/exportDatasetToCloudStorage.py

```diff
@@ -0,0 +1,179 @@
+import flowservice
+import destinationinstanceservice
+import os
+import time
+from utils import Utils
+
+class ExportDatasetToCloudStorage:
+    global flow_conn
+    global dis_conn
+    global utils
+    global username
+    def __init__(self):
+        self.username = os.getlogin()
+        self.flow_conn = flowservice.FlowService()
+        self.dis_conn = destinationinstanceservice.DestinationInstanceService()
+        self.utils = Utils()
```

Comment:
Utility function should probably just be static functions so you don't need to create an instance of it just to call these utility functions.

Reply:
I have made them static, thanks for the feedback, totally make sense.